src/test/CURSOR_XY_PROFILE.test.ts

```diff
@@ -304,7 +304,7 @@ describe("CURSOR_XY_PROFILE tests", () => {
             ].map(
                 function([fileID, point, assertPoint, profileLen, value]: 
                         [number, {x: number, y: number}, {x: number, y: number}, {x: number, y: number}, number]) {
-                    test(`assert the fileID "${fileID}" returns: Value=${value}, Profile length={${profileLen.x}, ${profileLen.y}}, Point={${assertPoint.x}, ${assertPoint.y}} as acquireing {${point.x}, ${point.y}}.`, 
+                    test(`assert the fileID "${fileID}" returns: Value=${value}, Profile length={${profileLen.x}, ${profileLen.y}}, Point={${assertPoint.x}, ${assertPoint.y}} as pointing {${point.x}, ${point.y}}.`, 
```

Comment:
"pointing" means something different. Rather just leave that word out, the meaning is clearer that way.